### PR TITLE
Remove selected nursery from destination nurseries

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

File: src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx [hunk 1]
```diff
@@ -100,6 +101,14 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
     }));
   };
 
+  useEffect(() => {
+    const allNurseries = getAllNurseries(organization);
+    const destinantionNurseries = allNurseries.filter((nur) => nur.id.toString() !== selectedNursery);
```
Comment (karthikbtf) [inline src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx:106]: typo destinationNurseries
also we could rename `nur` to `nursery`? :) more readable..

File: src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx [hunk 2]
```diff
@@ -231,10 +240,7 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                       id='destinationFacilityId'
                       label={strings.TO_NURSERY_REQUIRED}
                       selectedValue={localRecords[0].destinationFacilityId?.toString()}
-                      options={getAllNurseries(organization).map((nursery) => ({
-                        label: nursery.name,
-                        value: nursery.id.toString(),
-                      }))}
+                      options={destinationNurseriesOptions}
```
Comment (karthikbtf) [inline src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx:243]: should this be a state or can it be a computed function like we do on line #230 
either is fine, just looking at consistency

**Review by karthikbtf — COMMENTED** [review]
looks good, some comments

**Review by karthikbtf — APPROVED** [review]
Looks good otherwise, feel free to merge after following up 🎉

File: src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx (hunk 2 above)
Comment (constanzauanini) [reply src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx:243]: The problem here is that we need to change the options according to selectedNursery. I don't think the function will execute automatically every time that value change

**Pull request merged**